src/helpers/env.ts

```diff
@@ -19,5 +19,6 @@ export default cleanEnv(import.meta.env, {
   VITE_KETL_INVITES_BACKEND: str({
     default: KETL_INVITES_BACKEND,
   }),
+  VITE_SENTRY_DSN: str(),
```

Comment:
Don't forget to update env file in the 1Pass

Reply:
Ok